app/services/create_sample_data.rb

```diff
@@ -0,0 +1,22 @@
+##
+# A class that takes a user and creates two sample collections with
+# manifests
+class CreateSampleData
+  def initialize(user:, name:, description:, manifests:)
+    @user = user
+    @name = name
+    @description = description
+    @manifests = manifests
+  end
+
+  def save
+    collection = Collection.new(name: @name, description: @description, user: @user)
```

Comment:
You can to this more succinctly by simply calling `create` instead of `new` (and then you can drop the `collection.save` below.